openstack/clustering/v1/clusters/requests.go

```diff
@@ -46,3 +47,40 @@ func Create(client *gophercloud.ServiceClient, opts CreateOptsBuilder) (r Create
 
 	return
 }
+
+// ListOptsBuilder Builder.
+type ListOptsBuilder interface {
+	ToClusterListQuery() (string, error)
+}
+
+// ListOpts params
+type ListOpts struct {
+	Limit         int    `q:"limit"`
+	Marker        string `q:"marker"`
+	Sort          string `q:"sort"`
+	GlobalProject string `q:"global_project"`
```

Comment:
This looks like a bool value?

https://github.com/openstack/senlin/blob/e5bc16b3fea2f10dead365f983c5a937f23d72b0/senlin/api/openstack/v1/clusters.py#L58-L61

As well, if `false` is a valid value to pass, it should be `*bool`.